### ISSUE TITLE
Setup rultor

### ISSUE THREAD

**Issue by velo** [mpollmeier/orientdb-gremlin#92]
http://www.rultor.com/

Rultor gives on simple means to do releases...

That way, I could create releases as well =D

https://github.com/velo/maven-formatter-plugin/issues/16

**Comment by mpollmeier** [comment]
hmm, looks really cool. will have a look some time

**Comment by mpollmeier** [comment]
@rultor hello

**Comment by mpollmeier** [comment]
playing around in https://github.com/mpollmeier/orientdb-gremlin/pull/95

**Comment by mpollmeier** [comment]
Ok I spent enough time on this... I'm giving up. I'll keep the current state in branch `rultor` and revert some commits in master to get us back into a green state.
Here's what I tried and what went wrong, in case someone else wants to try fix rultor integration: 

With all the described changes to the pom as described in http://www.yegor256.com/2014/08/19/how-to-release-to-maven-central.html WITHOUT inheriting from some parent pom, rultor did not find the provided gpg keys. I basically had the same issue as in https://github.com/yegor256/rultor/issues/915

As suggested in https://github.com/yegor256/rultor/issues/915 I then used jcapi-parent as the parent pom as it configures a few more things that rultor needs. Then it did find the gpg keys, but as it configures every plugin under the sun I then ran into loads of problems with travis (e.g. i had to unset some java compiler flags, reconfigure the maven failsafe forkCount etc.) and now I'm down to a NullPointerException that I have no idea how to fix.
